server/chat-commands.js

```
@@ -676,7 +676,7 @@ const commands = {
 		if (!this.canTalk()) return;
 
 		let message = Chat.namefilter(target, user, true);
-		if (!message && target) return user.popup("Your status contains a banned word.");
+		if (!message && toID(target)) return user.popup("Your status contains a banned word.");
```

Should probably be more like `target.trim()`?

---

toID also catches statuses made up entirely of symbols (eg, ``/away .``)

---

Yes, but statuses made up entirely of banned symbols are different from calling the function without a status.

---

Like, I don't want a message consisting entirely of swastikas to be allowed, just because it doesn't contain a letter.

---

The status full of swastikas is still going through the namefilter and getting cleared, it just fails silently instead of telling the user their status is banned.

---

It seems like it shouldn't fail silently.